solrman/smstorage/zk_storage.go

```diff
@@ -311,6 +311,21 @@ func (s *ZkStorage) SetSplitsDisabled(disabled bool) error {
 	return nil
 }
 
+func (s *ZkStorage) GetSplitsDisabledTime() time.Time {
+	path := s.disableSplitsPath()
+	exists, stat, err := s.conn.Exists(path)
+	if err != nil {
+		s.logger.Errorf("could not check the splits disabled at %s in ZK: %s", path, err)
+		return time.Time{}
+	}
+
+	if exists {
+		return time.Unix(stat.Ctime/1000, 0)
```

Comment:
Are we converting this to seconds here?

Reply:
Yes, added a comment to clarify

Reply:
Correction, we convert to seconds for the conversion to a Go Time struct since that is what time.Unix expects. However, the returned object is a Go Time struct which can be converted used as a variety of time granularities. I've updated the comments to reflect this.